scripts/install_vim.sh

```diff
@@ -1,88 +1,186 @@
 #!/bin/bash
 
 set -e
+set -x
 
 bail() {
   echo "$@"
   exit 1
 }
 
 init_vars() {
+  VIM=vim
```

Comment:
I think this should be `FLAVOR` or something.  It seems a little easy to lose track of the variable because the characters "vim" are all over.  WDYT?